### ISSUE TITLE
[BUG] Postgres does not create `nostr_ts_relay`

### ISSUE THREAD

**Issue by bennyhodl** [Cameri/nostream#139]
**Describe the bug**
When using the docker start command, the database does not start because the database is not created.

**To Reproduce**
1. `npm run docker:compose:start`
2. `docker logs db`

**Expected behavior**
Database to be created

**Screenshots**

**System (please complete the following information):**
 - OS: Ubuntu
 - Platform: docker]

**Logs**
```
2023-01-14 21:56:33.605 UTC [477] FATAL:  database "nostr_ts_relay" does not exist
2023-01-14 21:56:38.706 UTC [484] FATAL:  database "nostr_ts_relay" does not exist
2023-01-14 21:56:44.063 UTC [492] FATAL:  database "nostr_ts_relay" does not exist
2023-01-14 21:56:49.182 UTC [499] FATAL:  database "nostr_ts_relay" does not exist
2023-01-14 21:56:54.289 UTC [506] FATAL:  database "nostr_ts_relay" does not exist
2023-01-14 21:56:59.469 UTC [514] FATAL:  database "nostr_ts_relay" does not exist
2023-01-14 21:57:04.609 UTC [521] FATAL:  database "nostr_ts_relay" does not exist
2023-01-14 21:57:09.728 UTC [528] FATAL:  database "nostr_ts_relay" does not exist
2023-01-14 21:57:14.849 UTC [535] FATAL:  database "nostr_ts_relay" does not exist
2023-01-14 21:57:19.952 UTC [542] FATAL:  database "nostr_ts_relay" does not exist
2023-01-14 21:57:25.075 UTC [550] FATAL:  database "nostr_ts_relay" does not exist
2023-01-14 21:57:30.216 UTC [557] FATAL:  database "nostr_ts_relay" does not exist
```

**Additional context**
As well script doesn't start `nostr-relay`

**Comment by Cameri** [comment]
@bennyhodl  Are you on the latest release or are you on the `main` branch?

**Comment by Cameri** [comment]
Try deleting the `.nostr` folder inside the project:
`rm -rf <project>/.nostr`
And running `./scripts/start` again.

Also, could you provide the full logs?

**Comment by bennyhodl** [comment]
Was on `main` tested with release 1.16 as well. 
```
Attaching to cache, db, migrations, nostr-ts-relay
cache           | 1:C 15 Jan 2023 00:53:33.221 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
cache           | 1:C 15 Jan 2023 00:53:33.223 # Redis version=7.0.5, bits=64, commit=00000000, modified=0, pid=1, just started
cache           | 1:C 15 Jan 2023 00:53:33.224 # Configuration loaded
cache           | 1:M 15 Jan 2023 00:53:33.226 # Server initialized
cache           | 1:M 15 Jan 2023 00:53:33.231 # WARNING overcommit_memory is set to 0! Background save may fail under low memory condition. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
db              | 
db              | PostgreSQL Database directory appears to contain a database; Skipping initialization
db              | 
db              | 2023-01-15 00:53:33.368 UTC [1] LOG:  starting PostgreSQL 15.1 (Debian 15.1-1.pgdg110+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 64-bit
db              | 2023-01-15 00:53:33.369 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
db              | 2023-01-15 00:53:33.370 UTC [1] LOG:  listening on IPv6 address "::", port 5432
db              | 2023-01-15 00:53:33.380 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db              | 2023-01-15 00:53:33.388 UTC [27] LOG:  database system was shut down at 2023-01-15 00:52:46 UTC
db              | 2023-01-15 00:53:33.400 UTC [1] LOG:  database system is ready to accept connections
db              | 2023-01-15 00:53:38.421 UTC [37] FATAL:  database "nostr_ts_relay" does not exist
db              | 2023-01-15 00:53:43.638 UTC [45] FATAL:  database "nostr_ts_relay" does not exist
migrations      | 
migrations      | added 36 packages, and audited 37 packages in 4s
migrations      | 
migrations      | 3 packages are looking for funding
migrations      |   run `npm fund` for details
migrations      | 
migrations      | 1 high severity vulnerability
migrations      | 
migrations      | To address all issues, run:
migrations      |   npm audit fix --force
migrations      | 
migrations      | Run `npm audit` for details.
migrations      | npm notice 
migrations      | npm notice New major version of npm available! 8.19.3 -> 9.3.0
migrations      | npm notice Changelog: <https://github.com/npm/cli/releases/tag/v9.3.0>
migrations      | npm notice Run `npm install -g npm@9.3.0` to update!
migrations      | npm notice 
db              | 2023-01-15 00:53:45.684 UTC [46] FATAL:  database "nostr_ts_relay" does not exist
migrations      | database "nostr_ts_relay" does not exist
migrations      | error: database "nostr_ts_relay" does not exist
migrations      |     at Parser.parseErrorMessage (/code/node_modules/pg-protocol/dist/parser.js:287:98)
migrations      |     at Parser.handlePacket (/code/node_modules/pg-protocol/dist/parser.js:126:29)
migrations      |     at Parser.parse (/code/node_modules/pg-protocol/dist/parser.js:39:38)
migrations      |     at Socket.<anonymous> (/code/node_modules/pg-protocol/dist/index.js:11:42)
migrations      |     at Socket.emit (node:events:513:28)
migrations      |     at addChunk (node:internal/streams/readable:324:12)
migrations      |     at readableAddChunk (node:internal/streams/readable:297:9)
migrations      |     at Readable.push (node:internal/streams/readable:234:10)
migrations      |     at TCP.onStreamRead (node:internal/stream_base_commons:190:23)
migrations exited with code 1
service "migrations" didn't completed successfully: exit 1
```

**Comment by Cameri** [comment]
> Was on `main` tested with release 1.16 as well.
> 
> ```
> Attaching to cache, db, migrations, nostr-ts-relay
> cache           | 1:C 15 Jan 2023 00:53:33.221 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
> cache           | 1:C 15 Jan 2023 00:53:33.223 # Redis version=7.0.5, bits=64, commit=00000000, modified=0, pid=1, just started
> cache           | 1:C 15 Jan 2023 00:53:33.224 # Configuration loaded
> cache           | 1:M 15 Jan 2023 00:53:33.226 # Server initialized
> cache           | 1:M 15 Jan 2023 00:53:33.231 # WARNING overcommit_memory is set to 0! Background save may fail under low memory condition. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
> db              | 
> db              | PostgreSQL Database directory appears to contain a database; Skipping initialization
> db              | 
> db              | 2023-01-15 00:53:33.368 UTC [1] LOG:  starting PostgreSQL 15.1 (Debian 15.1-1.pgdg110+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 64-bit
> db              | 2023-01-15 00:53:33.369 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
> db              | 2023-01-15 00:53:33.370 UTC [1] LOG:  listening on IPv6 address "::", port 5432
> db              | 2023-01-15 00:53:33.380 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
> db              | 2023-01-15 00:53:33.388 UTC [27] LOG:  database system was shut down at 2023-01-15 00:52:46 UTC
> db              | 2023-01-15 00:53:33.400 UTC [1] LOG:  database system is ready to accept connections
> db              | 2023-01-15 00:53:38.421 UTC [37] FATAL:  database "nostr_ts_relay" does not exist
> db              | 2023-01-15 00:53:43.638 UTC [45] FATAL:  database "nostr_ts_relay" does not exist
> migrations      | 
> migrations      | added 36 packages, and audited 37 packages in 4s
> migrations      | 
> migrations      | 3 packages are looking for funding
> migrations      |   run `npm fund` for details
> migrations      | 
> migrations      | 1 high severity vulnerability
> migrations      | 
> migrations      | To address all issues, run:
> migrations      |   npm audit fix --force
> migrations      | 
> migrations      | Run `npm audit` for details.
> migrations      | npm notice 
> migrations      | npm notice New major version of npm available! 8.19.3 -> 9.3.0
> migrations      | npm notice Changelog: <https://github.com/npm/cli/releases/tag/v9.3.0>
> migrations      | npm notice Run `npm install -g npm@9.3.0` to update!
> migrations      | npm notice 
> db              | 2023-01-15 00:53:45.684 UTC [46] FATAL:  database "nostr_ts_relay" does not exist
> migrations      | database "nostr_ts_relay" does not exist
> migrations      | error: database "nostr_ts_relay" does not exist
> migrations      |     at Parser.parseErrorMessage (/code/node_modules/pg-protocol/dist/parser.js:287:98)
> migrations      |     at Parser.handlePacket (/code/node_modules/pg-protocol/dist/parser.js:126:29)
> migrations      |     at Parser.parse (/code/node_modules/pg-protocol/dist/parser.js:39:38)
> migrations      |     at Socket.<anonymous> (/code/node_modules/pg-protocol/dist/index.js:11:42)
> migrations      |     at Socket.emit (node:events:513:28)
> migrations      |     at addChunk (node:internal/streams/readable:324:12)
> migrations      |     at readableAddChunk (node:internal/streams/readable:297:9)
> migrations      |     at Readable.push (node:internal/streams/readable:234:10)
> migrations      |     at TCP.onStreamRead (node:internal/stream_base_commons:190:23)
> migrations exited with code 1
> service "migrations" didn't completed successfully: exit 1
> ```

Stop everything with
`./scripts/stop`

Then delete your existing postgres data folder:
`cd nostream` and then
`rm -rf .nostr/data`

Start again with:
`./scripts/start`

**Comment by bennyhodl** [comment]
Thanks, that worked!
